### PR TITLE
fixed lint compile time errors

### PR DESCRIPTION
**Pull request by lessthantrue**
When I pulled in a fresh copy of the new master branch after lint was added, I got a lot of compiler errors. If anyone else is dealing with those, this should fix them.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -15,16 +15,16 @@ window.onload = function () {
...
     if($_GET.id != undefined){
         getFromMyJSON($_GET.id);
     }
 };
 function GETfromUrl(){
     return location.search.substr(1).split("&").reduce(function(o,i){
-        u = decodeURIComponent;
-        [k,v] = i.split("=");
-        o[u(k)] = v&&u(v);
+        var u = decodeURIComponent;
```
Comment (Zeragamba) [inline js/script.js]: Would be nice to have something a bit more descriptive the "o", "u", and "i"

File: js/script.js [hunk 2]
```diff
@@ -245,11 +245,11 @@ function getFromMyJSON(id){
             var resp = JSON.parse(http.responseText);
             createEntitiesFromJSON(resp);
         }
-    }
+    };
     http.send();
 }
 
-function savebtn() {
+window['savebtn'] = function () {
```
Comment (Zeragamba) [inline js/script.js]: `window.savebtn` would work here as well

File: js/script.js [hunk 3]
```diff
@@ -258,12 +258,11 @@ function savebtn() {
         sendToMyJSON(jsonstring);
     }
 
-}
-
-window.closebtn = function () {
-    document.getElementById("blueprint").style.display = "none";
 };
 
+function closebtn() {
```
Comment (Zeragamba) [inline js/script.js]: I think this is used from the html. If it is it should be added to the window obiect.

**Review by Zeragamba — CHANGES_REQUESTED** [review]

**Review by Zeragamba — APPROVED** [review]

**Comment by SpyMaster356** [comment]
@lessthantrue can you push a new commit (edit the README or something) to trigger a Travis build?

**Comment by Zeragamba** [comment]
[fixes #40]

**Pull request merged**